localization/yabloc/yabloc_pose_initializer/CMakeLists.txt

```diff
@@ -18,20 +18,6 @@ find_package(Sophus REQUIRED)
 # Download DNN model
 include(download.cmake)
 
-# ===================================================
-# Clear ${PYTHON_EXECUTABLE} defined by mrt_cmake_module so that rosidl_generate_interfaces can look for it properly
-unset(PYTHON_EXECUTABLE)
-message(STATUS "PYTHON_EXECUTABLE: ${PYTHON_EXECUTABLE}")
-
-# ===================================================
-# Service
-rosidl_generate_interfaces(${PROJECT_NAME}
-  "srv/SemanticSegmentation.srv"
```

Comment:
Please remove SemanticSegmentation.srv related documents from Readme

Reply:
Thanks for your reviewing :heart: I fixed it in [cfe971c](https://github.com/autowarefoundation/autoware.universe/pull/5025/commits/cfe971c9511234c3e418efd4d8536019680e6931)